src/main/java/org/primeframework/mvc/security/BaseJWTRefreshTokenCookiesUserLoginSecurityContext.java

```diff
@@ -288,6 +292,8 @@ private Tokens refreshJWT(Tokens tokens) {
       body.put("client_secret", List.of(oauthConfiguration.clientSecret));
     }
 
+    body.putAll(additionalTokenCallParams());
```

Comment:
Should we just add `tenantId` to the `oauthConfiguration` object and then do something like:

```
body.put("tenantId", oauthConfiguration.tenantId);
```

When we call `oauthConfiguration()` on line `273` we have the option to dynamically set `tenantId` so in theory you would know the `tenantId` in time? 

Or - since `tenantId` isn't standard, and we just need it in FusionAuth, we could add this optional map to `oauthConfiguration` so the impl can add whatever body parameters it wants but keep it in the existing config object?